src/Runner.Worker/GitHubContext.cs

```diff
@@ -32,6 +32,7 @@ public sealed class GitHubContext : DictionaryContextData, IEnvironmentContextDa
             "sha",
             "workflow",
             "workspace",
+            "attempt",
```

Comment:
what's the value look like for `GITHUB_ATTEMPT`? `bool`/`int`/`string`

Reply:
`int` - is there a change needed elsewhere?

Reply:
We might need to change line 42 since it checks `data.Value is StringContextData`, I forgot whether an `int` will be sent down as String or not. That's the reason I ask you whether you get it working E2E in the dev environment.

Reply:
Yeah I'm still working on that.  I'm struggling to get all three different repos to the right state without everything falling apart along the way 😞